.github/workflows/ci.yml

```
@@ -218,6 +218,54 @@ jobs:
         run: npm run js-api-spec -- --sassPackage ../embedded-host-node --sassSassRepo ../language
         working-directory: sass-spec
 
+  sass_spec_js_browser:
+    name: "JS API Tests | Pure JS | Dart ${{ matrix.dart_channel }} | Browser"
```

To make this a little easier to differentiate in the web UI, let's make it

```suggestion
    name: "JS API Tests | Browser | Dart ${{ matrix.dart_channel }}"
```

since "browser" implies "pure JS" anyway.